app/media/Dockerfile

```diff
@@ -19,4 +19,4 @@ ENV MEDIA_UPLOAD_LOCATION="/uploads"
 ARG version
 ENV VERSION=$version
 
-CMD cd src && uwsgi --http 0.0.0.0:5000 --wsgi-file media/server.py --callable app
+CMD cd src && uwsgi --http 0.0.0.0:5001 --wsgi-file media/server.py --callable app
```

Comment:
I was a liiiiitle unsure about this one so just flagging it. Since it says media/server and I changed that env var I think I wanna change this too?

Reply:
Yep! This tells the media server to actually run on that port